demos/guide/README.md

```diff
@@ -51,25 +51,16 @@ The tls-client-non-block demo can be run in exactly the same way. Just replace
 Running the QUIC Demos
 ----------------------
 
-The QUIC demos can be run in a very similar way to the TLS demos. However, a
-different server implementation will need to be used.
+The QUIC demos can be run in a very similar way to the TLS demos.
 
-The OpenSSL source distribution includes a test QUIC server implementation for
-use with the demos. Note that, although this server does get built when building
-OpenSSL from source, it does not get installed via "make install". After
-building OpenSSL from source you will find the "quicserver" utility in the
-"util" sub-directory of the top of the build tree. This server utility is not
-suitable for production use and exists for test purposes only. It will be
-removed from a future version of OpenSSL.
+While in the demos directory the QUIC server can be run like this:
 
-While in the demos directory the quic server can be run like this:
-
-./../util/quicserver localhost 4443 servercert.pem serverkey.pem
+./quic-server-block 4443 ./chain.pem pkey.pem
 
 The QUIC demos can then be run in the same was as the TLS demos. For example
 to run the quic-client-block demo:
 
-SSL_CERT_FILE=rootcert.pem LD_LIBRARY_PATH=../.. ./quic-client-block localhost 4443
```

Comment:
Why are we removing LD_LIBRARY_PATH here?

Reply:
Ah, that should be reverted

Reply:
quic-client-block doesn't need to specify LD_LIBRARY_PATH to work, but I can revert it back to the original